code/style.css

```diff
@@ -2,6 +2,184 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+* {
```

Comment:
I think this file needs some love to it. Not the functionality in the CSS, more structure wise. There is some un needed empty rows and the indention isn't the same everywhere. Those things can seems to be nothing and not important, but it really is! Good looking code is as important as the functionality. And it is something that developer recruiter will look at so it's good to work on it from the start 😀